src/inspector/models/notation/notes/beams/beamsettingsmodel.cpp

```diff
@@ -193,11 +193,16 @@ void BeamSettingsModel::setBeamHeight(const qreal left, const qreal right)
 
 void BeamSettingsModel::updateFeatheringMode(const qreal left, const qreal right)
 {
+    BeamTypes::FeatheringMode newFeathering;
```

Comment:
You should assign a default value. I would suggest rewriting this code as follows:

```
BeamTypes::FeatheringMode newFeathering = BeamTypes::FeatheringMode::FEATHERING_NONE;

if (left != right) {
    ...
}

if (newFeathering != m_featheringMode) {
    m_featheringMode = newFeathering;
    emit featheringModeChanged(m_featheringMode);
}

```